press/utils.py

```diff
@@ -0,0 +1,13 @@
+__all__ = (
+    'convert_version_to_legacy_version',
+)
+
+
+def convert_version_to_legacy_version(version):
+    """Converts a version to a legacy version.
+
+    :param version: content's major and minor version
+    :type version: tuple of int
+
+    """
+    return '1.{}'.format(version[0])
```

Comment:
FYI there are about 1000 old modules (last edited 2015) for which this is not true (they're numbered 2.X, starting at 2.0). Probably best to ignore them for purposes of this code, though - they're not openstax books, so are out of scope for batch for now.